src/clients/inst_scc.rb

```diff
@@ -304,12 +306,14 @@ def start_workflow
           next:  "register_addons"
         },
         "register_addons"        => {
-          abort: :abort,
-          next:  "update_autoyast_config"
+          abort:      :abort,
+          extensions: "select_addons",
+          next:       "update_autoyast_config"
         },
         "reregister_addons"      => {
-          abort: :abort,
-          next:  "check"
+          abort:      :abort,
+          extensions: "select_addons",
```

Comment:
Thanks for that point!! sure i mean select_addons_rereg, i wii fix and test it manually